lib/src/ui/common/util/mappers.dart

```diff
@@ -0,0 +1,18 @@
+import 'package:allthenews/src/ui/pages/home/news/primary_news/primary_news_list_entity.dart';
+import 'package:allthenews/src/ui/pages/home/news/secondary_news/secondary_news_list_entity.dart';
+
+extension PrimaryNewsEntityMapper on List<PrimaryNewsListEntity> {
+  List<SecondaryNewsListEntity> mapToSecondaryNewsListEntities() {
```

Comment:
to taka mega drobnostka, ale usunąłbym słowo map, bo wszystkie metody które mapują z jednego typu mają raczej konwencję `toX`, gdzie X to typ np. toInt, toString, toList itd.